manual/src/main/asciidoc/user-guide/provisioning.adoc

```diff
@@ -123,16 +123,25 @@ To prevent this behaviour, you can specify features as boot features.
 
 ==== Boot features
 
-You can describe some features as boot features. A boot feature will be automatically install by Apache Karaf, even if it has
-not been previously installed using `feature:install` or FeatureMBean.
+A boot feature is automatically installed by Apache Karaf, even if it has not been previously installed using `feature:install` or FeatureMBean.
 
-Apache Karaf features configuration is located in the `etc/org.apache.karaf.features.cfg` configuration file.
+The Apache Karaf features configuration is located in the `etc/org.apache.karaf.features.cfg` configuration file.
 
 This configuration file contains the two properties to use to define boot features:
 
 * `featuresRepositories` contains a list (comma-separated) of features repositories (features XML) URLs.
 * `featuresBoot` contains a list (comma-separated) of features to install at boot.
 
+========
```

Comment:
I think it's worth to mention that it's possible to stop Karaf, update `featuresBoot` and remove the data folder. It's another way to cleanup the `featuresBoot`.